src/pages/en/guides/typescript.md

```diff
@@ -135,3 +137,25 @@ To see type errors in your editor, please make sure that you have the [Astro VS
 
 📚 Read more about [`.ts` file imports](/en/guides/imports/#typescript) in Astro.  
 📚 Read more about [TypeScript Configuration](https://www.typescriptlang.org/tsconfig/).
+
+## Troubleshooting
+
+### Typing multiple JSX frameworks at the same time
```

Comment:
```suggestion
### Errors Typing multiple JSX frameworks at the same time
```
Making this read more like a problem to fix, so that we can make it a similar form to our Troubleshooting page